config/jest-config-carbon/index.js

```diff
@@ -20,7 +20,7 @@ module.exports = {
     '<rootDir>/**/*-(spec|test).js?(x)',
   ],
   transform: {
-    '^.+\\.(js|jsx)$': require.resolve('./transform/jsTransform.js'),
+    '^.+\\.(mjs|cjs|js|jsx)$': require.resolve('./transform/jsTransform.js'),
```

Comment:
Just out of curiosity, what are `mjs` and `cjs`?

Reply:
`.mjs` is an extension in Node v14 and newer for ESM support in Node.js! `.cjs` is for interop between the two